canonical-web-and-design/ubuntu.com
# Security series webinar

## Done

- Build takeover
- Build even page at `/security-series`
- Add takeover to `takeovers/index.html`
- Add takeover to `index.html`
- Build [social banner](https://tools.demos.haus/?title=Ubuntu+Security%3A+Security+and+compliance&subtitle=The+full+Ubuntu+security+story.+Learn+how+our+teams+are+securing+all+your+open+source%2C+from+cloud+to+edge.&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2F1899df43-ubuntu-advantage-circular_white.svg&background=grad#banner-form-modal)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security-series
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the site matches the [design](https://user-images.githubusercontent.com/65166314/93861516-ed194c80-fcc0-11ea-933e-a87dca1a7375.png) and the [brief](https://docs.google.com/document/d/1A7JDBRZ_tuEvteE1pVA9_rV-oJUq1_DuT7GmO6zVCXM/edit#)
- Check takeover matches [design](https://github.com/canonical-web-and-design/web-squad/issues/3114) and [brief](https://docs.google.com/document/d/1duufuOPuy6asv-i9qtGNs3y1VLRO67GYaOfwE9OGQmw/edit)


## Issue / Card

Fixes #8184 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/94280215-0fbb9780-ff45-11ea-9f9c-f32accde0249.png)
